repos/system_upgrade/el8toel9/actors/checkcustomnetworkscripts/libraries/customnetworkscripts.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from leapp import reporting
+
+CUSTOM_NETWORK_SCRIPTS = [
+    "/sbin/ifup-local",
+    "/sbin/ifup-pre-local",
+    "/sbin/ifdown-local",
+    "/sbin/ifdown-pre-local",
+]
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
```

Comment:
Could you move DOC_URL to the same level as CUSTOM_NETWORK_SCRIPTS constant?

Reply:
I moved it to the same level as  CUSTOM_NETWORK_SCRIPTS constant.